src/lib/API/api.js

```diff
@@ -152,7 +152,7 @@ class API {
    * @param {object} verificationData
    * @param {string} verificationData.code
    */
-  verifyEmail(verificationData: { code: string }): Promise<$AxiosXHR<any>> {
+  verifyEmail(verificationData: { code: integer }): Promise<$AxiosXHR<any>> {
```

Comment:
like @fernandomg commented, we should treat code as string. ie convert it .toString().